sql/hive/benchmarks/OrcReadBenchmark-results.txt

```diff
@@ -3,220 +3,220 @@ SQL Single Numeric Column Scan
-Native ORC Vectorized                               101            130          24         10.4          96.2       1.5X
-Hive built-in ORC                                  1477           1494          25          0.7        1408.7       0.1X
+Native ORC MR                                       200            239          35          5.2         190.5       1.0X
+Native ORC Vectorized                               145            174          23          7.3         137.8       1.4X
+Hive built-in ORC                                  1809           1945         193          0.6        1725.0       0.1X
 
 OpenJDK 64-Bit Server VM 1.8.0_312-b07 on Linux 5.11.0-1022-azure
-Intel(R) Xeon(R) Platinum 8272CL CPU @ 2.60GHz
+Intel(R) Xeon(R) CPU E5-2673 v4 @ 2.30GHz
 Single Column Scan from 300 columns:      Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 ------------------------------------------------------------------------------------------------------------------------
-Native ORC MR                                       191            227          29          5.5         182.4       1.0X
-Native ORC Vectorized                               135            153          18          7.7         129.2       1.4X
-Hive built-in ORC                                  2085           2085           0          0.5        1988.1       0.1X
+Native ORC MR                                       278            336          52          3.8         264.7       1.0X
+Native ORC Vectorized                               209            232          21          5.0         199.0       1.3X
+Hive built-in ORC                                  2679           2828         211          0.4        2554.8       0.1X
 
 
 ================================================================================================
 Struct scan
 ================================================================================================
 
 OpenJDK 64-Bit Server VM 1.8.0_312-b07 on Linux 5.11.0-1022-azure
-Intel(R) Xeon(R) Platinum 8272CL CPU @ 2.60GHz
+Intel(R) Xeon(R) CPU E5-2673 v4 @ 2.30GHz
 Single Struct Column Scan with 10 Fields:  Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 ------------------------------------------------------------------------------------------------------------------------
-Native ORC MR                                      1126           1149          33          0.9        1073.7       1.0X
-Native ORC Vectorized                              1136           1141           7          0.9        1083.4       1.0X
-Hive built-in ORC                                   589            595           8          1.8         561.4       1.9X
+Native ORC MR                                       363            427          78          2.9         346.2       1.0X
+Native ORC Vectorized                               375            442          88          2.8         357.6       1.0X
```

Comment:
just FYI @bersprockets - for vectorized scan of nested column, I have fixed the benchmark to enable vectorization - https://github.com/apache/spark/commit/4a2ba5b22e84fc79b44604c60320aa5ae679e13a .